webapp-django/crashstats/signature/static/signature/css/signature_report.less

```diff
@@ -118,10 +108,21 @@ a.crash-id {
     }
 }
 
-#graph-panel {
+.graph-panel {
     .controls {
         .channels-list {
             width: 30%;
         }
     }
 }
+
+.tab-inner-panel .delete {
+    background-image: url('../../img/3rdparty/silk/cross.png');
+    display: inline-block;
+    float: right;
+    height: 16px;
+    margin-top: 10px;
+    padding: 4px;
+    text-indent: -9999px;
+    width: 16px;
+}
```

Comment:
There is a rule in ``screen.less`` that overwrites the background of this element on hover, makes it ugly. Not a big thing though, we might just do a quick follow up if that's the only problem I find.